lib/config.go

```diff
@@ -28,21 +30,29 @@ var SaveConfig = func() error {
 
 // LoadConfig loads the global default configuration
 func LoadConfig(streams ioes.IOStreams, path string) (err error) {
-	var cfg *config.Config
-	cfg, err = config.ReadFromFile(path)
+	var data []byte
```

Comment:
maybe the changes here could be moved into `config.ReadFromFile` instead of throwing it out, that way we have  a clean package API that properly calls fill_struct.

but I think that would create a circular dependency between `base` & `config`, so we'd need to make fill struct it's own package for that to work. If that's right, no need to do this now.

Reply:
My first approach was doing this in config.ReadFromFile, but yeah it is a circular dependency, because config/ is actually below base/. This implies that fill_struct may need to move to its own package, but I agree we should do it later.